### PR TITLE
showed posts in separate component for a hooks using to load more

### PR DESCRIPTION
**Pull request by bezverkhnii**

### CODE REVIEW

File: src/components/allNews/AllNews.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use client'
+import React, { useState } from 'react'
+import NewsBox from '@/components/newsBox/NewsBox'
+import { formatDate } from '@/utils/formatters'
+import styles from './allNews.module.css'
+
+const AllNews = ({ posts }) => {
```
Comment (apyrkh) [inline src/components/allNews/AllNews.jsx:7]: please rename to `NewsFeed`

File: src/app/news/page.jsx [hunk 2]
```diff
@@ -1,60 +1,17 @@
 import { getPagePosts } from '@server/db/services/pagePostService'
 import { React } from 'react'
 import styles from './page.module.css'
-import NewsBox from '@/components/newsBox/NewsBox'
+// import NewsBox from '@/components/newsBox/NewsBox'
 import Image from 'next/image'
 import { allNews } from '@/constants/data'
 import { getText } from '@/utils/textUtils'
+import AllNews from '@/components/allNews/AllNews'
+// import { formatDate } from '@/utils/formatters'
```
Comment (apyrkh) [inline src/app/news/page.jsx:9]: clean up

File: src/app/news/page.jsx [hunk 3]
```diff
@@ -1,60 +1,17 @@
 import { getPagePosts } from '@server/db/services/pagePostService'
 import { React } from 'react'
 import styles from './page.module.css'
-import NewsBox from '@/components/newsBox/NewsBox'
+// import NewsBox from '@/components/newsBox/NewsBox'
```
Comment (apyrkh) [inline src/app/news/page.jsx:4]: clean up

File: src/components/allNews/AllNews.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  const [loadedPostsAmount, setloadedPostsAmount] = useState(6)
+  const loadedPosts = posts.slice(0, loadedPostsAmount)
+  const handleShowMore = () => {
+    setloadedPostsAmount((prev) => (prev += 6))
+  }
```
Comment (apyrkh) [inline src/components/allNews/AllNews.jsx:12]: this will be done in a different way, but for now let's keeps it as it is

**Review by apyrkh — CHANGES_REQUESTED** [review]

File: src/components/newsBox/newsBox.module.css [hunk 5]
```diff
@@ -11,7 +11,11 @@
 }
 
 .container:hover .title {
-  color: var(--secondary-color);
+  opacity: 0.6;
+}
+
+.container:hover {
+  border-color: rgba(211, 211, 211, 0.632);
```
Comment (apyrkh) [inline src/components/newsBox/newsBox.module.css:18]: will it work for both dark/light themes?

File: src/components/newsBox/newsBox.module.css [hunk 6]
```diff
@@ -39,6 +43,12 @@
 }
 
 .desc {
-  color: rgb(137, 136, 136);
+  opacity: 0.6;
+  font-weight: lighter;
+}
+
+.createdAt {
+  color: rgba(137, 136, 136, 0.662);
```
Comment (apyrkh) [inline src/components/newsBox/newsBox.module.css:51]: will it work for both dark/light themes?

**Review by apyrkh — CHANGES_REQUESTED** [review]

File: src/components/newsBox/newsBox.module.css (hunk 5 above)
Comment (bezverkhnii) [reply src/components/newsBox/newsBox.module.css]: implemented changes

File: src/components/newsBox/newsBox.module.css (hunk 6 above)
Comment (bezverkhnii) [reply src/components/newsBox/newsBox.module.css]: implemented changes

File: src/components/allNews/AllNews.jsx (hunk 4 above)
Comment (bezverkhnii) [reply src/components/allNews/AllNews.jsx:12]: okay

**Pull request merged**